monitoring/alerts.test.yaml

```diff
@@ -229,6 +231,105 @@ tests:
         eval_time: 28m
         exp_alerts: []
 
+  # QuotaMetricsNotAvailable (case with account quota)
+  ##################################################################################################
+  - name: Quota metrics not available (account quota)
+    interval: 1m
+    input_series:
+      - series: s3_cloudserver_quota_utilization_service_available{namespace="zenko",service="artesca-data-connector-s3api-metrics"}
+        values: 1+1x6 0+0x20 1+1x6
+      - series: s3_cloudserver_quota_accounts_count{namespace="zenko",job="artesca-data-ops-report-handler"}
```

Comment:
should also add test when `s3_cloudserver_quota_accounts_count` is not present (i.e. maybe add a few `undefined` at the beginning of the series)